src/Filter.js

```diff
@@ -119,9 +88,12 @@ function Filter({
     <div className="dq-main-container">
       <div className="dq-heading">
         <div className="dq-heading-main"></div>
-        <div data-testid="agg" className="dq-heading-total-rows">
-          Total rows: {total && total.toLocaleString()}
-        </div>
+        <Totalrows
+          newFilter={newFilter}
+          dataset={dataset}
+          setTotal={setTotal}
```

Comment:
This makes no sense - why `Totalrows` component needs all of these?

Reply:
the `newFilter` and `dataset` are used inside the query to get the total of rows and the `setTotal` is the state to set the total of rows because it's also used on the table container to handle the page navigation.